rpmbuild/copr_rpmbuild/automation/fedora_review.py

```diff
@@ -67,6 +86,18 @@ def _filter_results_directory(self):
         dstdir = os.path.join(self.resultdir, "fedora-review")
         os.makedirs(dstdir, exist_ok=True)
 
+        logfile = os.path.join(cachedir, "fedora-review.log")
+
+        # The fedora-review command failed to start so badly that it didn't even create a log file.
+        # Given how early that should happen, this merits investigation.
+        if not os.path.exists(logfile):
+            self.log.error("Can't find fedora-review log: %s", logfile)
+            self.log.error("Please raise a bug on https://github.com/fedora-copr/copr including a link to this build")
+            # If the log file doesn't exist, there's zero chance the results exist
+            return
+
+        os.rename(logfile, os.path.join(dstdir, "fedora-review.log"))
```

Comment:
We need to compress this file (likely on backend, `_compress_live_logs` should be named `_compress_logs` and it should cover even this file.  Would you mind filling issue, or fixing directly?

Reply:
See https://github.com/fedora-copr/copr/pull/2920/commits/a3b060fd8b39cc52b71f2a944eb35063f5be4ee4